hardware/panels.py

```diff
@@ -57,7 +72,7 @@ def exposure(self):
     def power(self):
         eff = 0.3 # default, if the temperature curve is not set for the sun
         if self.sun.regolith_temperature is not None: eff = self.pvEfficiency(self.sun.regolith_temperature)
-        power =  Panel.solarConstant*eff*np.select(self.condition_list, self.choice_list) * self.efficiency_mult
+        power =  Panel.solarConstant*eff*np.select(self.condition_list, self.choice_list)*self.pv_angle_corr * self.efficiency_mult
```

Comment:
Does this actually work? You're  calling it as a variable, not as a function? Or am I missing something?

Reply:
Yep. In line 37 the variable version is defined from the function version with the current sun coordinates:         self.pv_angle_corr = self.pv_angle_corr(sun.xyz). That was the way Maxim had the dot product and normal vector rotation defined, I think for simplicity of handling, so I followed that convention.

Reply:
Ok, but this is kinda super dangerous, because if nothing else interpreter would replace the function with a variable. Can you just say instead something like `self.pv_angle_corr_calculated = self.pv_angle_corr(sun.xyz)`?

Reply:
Good point. Variable name changed to dot_sun_corr since it's a correction to the sun-PV dot product, which is called dot_sun.